docs/content/tutorials/tutorial-batch-hadoop.md

```diff
@@ -0,0 +1,247 @@
+#druid.indexer.logs.type=file
+#druid.indexer.logs.directory=var/druid/indexing-logs
+
+# For HDFS:
+druid.indexer.logs.type=hdfs
+druid.indexer.logs.directory=/druid/indexing-logs
+
+```
+
+### Restart Druid cluster
+
+Once the Hadoop .xml files have been copied to the Druid cluster and the segment/log storage configuration has been updated to use HDFS, the Druid cluster needs to be restarted for the new configurations to take effect.
+
+If the cluster is still running, CTRL-C to terminate the `bin/supervise` script, and re-reun it to bring the Druid services back up.
+
+## Load batch data
+
+We've included a sample of Wikipedia edits from June 27, 2016 to get you started.
+
+To load this data into Druid, you can submit an *ingestion task* pointing to the file. We've included
+a task that loads the `wikiticker-2015-09-12-sampled.json.gz` file included in the archive. To submit
+this task, POST it to Druid in a new terminal window from the druid-#{DRUIDVERSION} directory:
+
+```bash
+curl -X 'POST' -H 'Content-Type:application/json' -d @quickstart/tutorial/wikipedia-index-hadoop.json http://localhost:8090/druid/indexer/v1/task
+```
+
+Which will print the ID of the task if the submission was successful:
+
+```bash
+{"task":"index_hadoop_wikipedia-hadoop_2018-06-09T21:30:32.802Z"}
+```
+
+To view the status of your ingestion task, go to your overlord console:
+[http://localhost:8090/console.html](http://localhost:8090/console.html). You can refresh the console periodically, and after
+the task is successful, you should see a "SUCCESS" status for the task.
+
+After your ingestion task finishes, the data will be loaded by historical nodes and available for
+querying within a minute or two. You can monitor the progress of loading your data in the
+coordinator console, by checking whether there is a datasource "wikipedia" with a blue circle
+indicating "fully available": [http://localhost:8081/#/](http://localhost:8081/#/).
+
+![Coordinator console](../tutorials/img/tutorial-batch-01.png "Wikipedia 100% loaded")
+
+## Querying your data
+
+Your data should become fully available within a minute or two after the task completes. You can monitor this process on 
+your Coordinator console at [http://localhost:8081/#/](http://localhost:8081/#/).
+
+Please follow the [query tutorial](../tutorial/tutorial-query.html) to run some example queries on the newly loaded data.
+
+## Cleanup
+
+If you wish to go through any of the other ingestion tutorials, you will need to shut down the cluster and reset the cluster state by removing the contents of the `var` directory under the druid package, as the other tutorials will write to the same "wikipedia" datasource.
+
+After finishing this tutorial, if you plan on doing other tutorials, you will need to revert the deep storage and task storage config back to local types in `quickstart/tutorial/conf/druid/_common/common.runtime.properties` and then restart the cluster.
```

Comment:
I redid this section to this:

```
This tutorial is only meant to be used together with the [query tutorial](../tutorial/tutorial-query.html). 

If you wish to go through any of the other tutorials, you will need to:
* Shut down the cluster and reset the cluster state by removing the contents of the `var` directory under the druid package.
* Revert the deep storage and task storage config back to local types in `quickstart/tutorial/conf/druid/_common/common.runtime.properties`
* Restart the cluster

This is necessary because the other ingestion tutorials will write to the same "wikipedia" datasource, and later tutorials expect the cluster to use local deep storage.
```